extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -92,13 +101,30 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
     return isInputLimitExceeded;
   };
 
+  private initSquire = (addLinks: boolean, removeExistingLinks = false) => {
```

Comment:
Maybe we can leave just `addLinks` parameter? As existing links should be removed only when `addLinks = false`

Reply:
Yeah, it was same with my thought and was my original code.
But below test fails which contains signature. As you know signature should start with two empty lines, like below but if we call `htmlSanitizeAndStripAllTags` it will trim first white spaces and therefore two empty lines in signature will be removed.

```


--
Regards,
```

https://flowcrypt.semaphoreci.com/workflows/4757d0a2-e83b-4a76-99e1-3d354b1bb63f/summary?pipeline_id=254256c3-c2b7-4106-bb30-9716f50e4e21&report_id=4ae71336-e44b-39bf-b9d2-752e234818a5

Reply:
Ah, got it, then no changes needed here 👍